source/includes/indexes/index-code-examples.scala

```diff
@@ -0,0 +1,57 @@
+// start-index-single
+val index = Indexes.ascending("<field name>")
+val observable = collection.createIndex(index)
+Await.result(observable.toFuture(), Duration(10, TimeUnit.SECONDS))
+// end-index-single
+
+// start-index-compound
+val index = Indexes.compoundIndex(
+    Indexes.descending("<field name 1>"), 
+    Indexes.ascending("<field name 2>")
+)
+val observable = collection.createIndex(index)
+Await.result(observable.toFuture(), Duration(10, TimeUnit.SECONDS))
+// end-index-compound
+
+// start-create-multikey-index
+val index = Indexes.ascending("<field name>")
+val observable = collection.createIndex(index)
+Await.result(observable.toFuture(), Duration(10, TimeUnit.SECONDS))
+// end-create-multikey-index
+
+// start-create-geospatial-index
+val observable = collection.createIndex(Indexes.geo2dsphere("<2d index>"))
+Await.result(observable.toFuture(), Duration(10, TimeUnit.SECONDS))
+// end-create-geospatial-index
+
+// start-drop-single-index
+val observable = collection.dropIndex("<index name>")
+Await.result(observable.toFuture(), Duration(10, TimeUnit.SECONDS))
+// end-drop-single-index
+
+// start-create-search-index
+val index = Document("mappings" -> Document("dynamic" -> true))
```

Comment:
S: I think a placeholder for the dynamic value is better, rather than speifying true or false:
```suggestion
val index = Document("mappings" -> Document("dynamic" -> <boolean value>))
```